src/main/java/wolox/training/mapper/BookMapper.java

```diff
@@ -0,0 +1,31 @@
+package wolox.training.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
+import org.mapstruct.ReportingPolicy;
+import wolox.training.dto.BookDTO;
+import wolox.training.integration.dto.Book;
+
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE, componentModel = "spring")
+public interface BookMapper {
+
+    @Mappings({
```

Comment:
MapStruct has an implicit mapper. If the name of the fields are the same, like in most of this case, the Mapping annotation is not needed.